sw/device/tock/meson.build

```diff
@@ -43,7 +47,7 @@ tock_raw = custom_target(
   output: target,
   console: true,
   build_always_stale: true,
-  build_by_default: false,
+  build_by_default: true,
```

Comment:
We shouldn't enable building Rust targets by default until we have toolchain requirements sorted out at the OT project level. This will break the build for those without rustup, including CI builds.

Reply:
Oh, absolutely. This is just a temporary change for work I am doing. I am not intending to merge this.